src/wallet.rs

```diff
@@ -247,19 +294,71 @@ pub fn gen_keypair_file(outfile: String) -> Result<String, Box<error::Error>> {
     Ok(serialized)
 }
 
+pub enum WalletRpcRequest {
+    ConfirmTransaction,
+    GetAccountInfo,
+    GetBalance,
+    GetFinality,
+    GetLastId,
+    GetTransactionCount,
+    RequestAirdrop,
+    SendTransaction,
+}
+impl WalletRpcRequest {
+    fn make_rpc_request(
+        &self,
+        rpc_addr: &SocketAddr,
+        id: u64,
+        params: Option<String>,
+    ) -> Result<Value, Box<error::Error>> {
+        let rpc_string = format!("http://{}", rpc_addr.to_string());
+        let jsonrpc = "2.0";
+        let method = match self {
+            WalletRpcRequest::ConfirmTransaction => "confirmTransaction",
+            WalletRpcRequest::GetAccountInfo => "getAccountInfo",
+            WalletRpcRequest::GetBalance => "getBalance",
+            WalletRpcRequest::GetFinality => "getFinality",
+            WalletRpcRequest::GetLastId => "getLastId",
+            WalletRpcRequest::GetTransactionCount => "getTransactionCount",
+            WalletRpcRequest::RequestAirdrop => "requestAirdrop",
+            WalletRpcRequest::SendTransaction => "sendTransaction",
+        };
+        let client = reqwest::Client::new();
+        let mut request: String = format!(
+            "{{\"jsonrpc\":\"{}\",\"id\":{},\"method\":\"{}\"",
```

Comment:
how about formatting this json template like this:
https://github.com/solana-labs/solana/blob/master/src/rpc.rs#L298-L306

Reply:
oh this might be fun to explore too: https://crates.io/crates/json_macros

Reply:
fyi, https://docs.serde.rs/serde_json/macro.json.html

Reply:
Yesterday, I was thinking the `json` macro didn't buy us much, but it is sooo much cleaner :)